app/models/client.rb

```diff
@@ -315,15 +358,23 @@ def unavailable(permanent:false, contact_id:nil)
     active_match = client_opportunity_matches.active.first
```

Comment:
This is a previous bug, but we should go ahead and fix it here.  We changed from active_match to `active_matches` relatively recently, looks like `def unavailable` missed the boat.  This should get converted to looping over `active_matches` and calling the bits inside the `if active_match.present?` clause.